packages/flutter/lib/src/cupertino/desktop_text_selection.dart

```diff
@@ -319,48 +319,6 @@ class _CupertinoDesktopTextSelectionToolbar extends StatelessWidget {
   }
 }
 
-// Positions the toolbar at [anchor] if it fits, otherwise moves it so that it
-// just fits fully on-screen.
-//
-// See also:
-//
-//   * [CupertinoDesktopTextSelectionToolbar], which uses this to position itself.
-//   * [TextSelectionToolbarLayoutDelegate], which does a similar layout for
-//     the mobile text selection toolbars.
-class _DesktopTextSelectionToolbarLayoutDelegate extends SingleChildLayoutDelegate {
```

Comment:
I moved this into its own file and made it public in order to share it with the Mac and Material desktop context menus.